types/slug/slug-tests.ts

```diff
@@ -19,6 +19,9 @@ slug("string", {}); // $ExpectType string
 
 slug("string", "replacement"); // $ExpectType string
 
+// eslint-disable-next-line no-trailing-spaces
```

Comment:
remove the eslint rule

Reply:
The test fails with or without the rule.

Reply:
I tested it without the rule. It works.

Reply:
He is trying to add functionality that is valid but the test is failing, hence the rule that he was trying to use to disable the whitespace check.

Reply:
<img width="629" alt="Snímka obrazovky 2021-11-29 o 21 50 45" src="https://user-images.githubusercontent.com/5845264/143941232-a03fd8e5-7cdb-493b-a8d1-2ff89783391d.png">

Reply:
I mean I see that it's passing for you... but the CI tests are failing both ways.